reference: update Tispark FAQ (#1553)
pingcap/docs: v2.1/reference/tispark.md

Q: What can I do if `java.sql.BatchUpdateException: Data Truncated` is returned when executing SQL statements using TiSpark?

A: This error occurs because the length of the data written exceeds the length of the data type defined by the database. You can check the field length and adjust it accordingly.

Q: Does TiSpark read Hive metadata by default?

> A: TiSpark searches for the Hive database by reading the Hive metadata in hive-site. If the search task fails, it searches for the TiDB database instead, by reading the TiDB metadata.

Reviewer: ```suggestion
A: TiSpark searches for the Hive database by reading the Hive metadata in hive-site by default. If the search task fails, it searches for the TiDB database instead, by reading the TiDB metadata.
```

> To disable this default behavior, do not configure the Hive metadata in hive-site.

Reviewer: This sentence is a little bit weird to me. How about:
```suggestion
If you do not need this default behavior, do not configure the Hive metadata in hive-site.
```
